x10/perpetual/trading_client/user_module.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+
+from x10.perpetual.accounts import AccountModel
+from x10.perpetual.trading_client.base_module import BaseModule
+from x10.utils.http import WrappedApiResponse, send_get_request
+
+
+class UserModule(BaseModule):
+    async def get_accounts(self) -> WrappedApiResponse[List[AccountModel]]:
+        url = self._get_url("/user/accounts")
+        return await send_get_request(
+            await self.get_session(),
+            url,
+            List[AccountModel],
+            api_key=self._get_api_key(),
```

Comment:
I don't think api key auth will work here, as the api key is only authenticated for a single account currently on backend. I think the `UserModule` will need to use the L1 auth schema

Reply:
Pls see the PR description -- it works (I think this is an error that should be fixed on a BE). While I do agree that a different auth schema should be used but don't have it right now.

Reply:
Let's do the following -- I will remove this module for now not to confuse SDK users while keeping the `AccountModel` which is required for transfers.